scripts/runFabtests.sh

```diff
@@ -125,35 +146,40 @@ function run_test {
 
 		wait $p2
 		ret2=$?
+
+		SO=$(cat $s_outp)
+		CO=$(cat $c_outp)
 	fi
 
-	if [ $ret1 != 0 -o $ret1 != 0 ]; then
+	if [ "$ret1" == "61" -a "$ret1" == "61" ]; then
+		printf "%-50s%10s\n" "$test_exe:" "Notrun"
+		skip_count+=1
+	
+	elif [ "$ret1" != "0" -o "$ret1" != "0" ]; then
 		if [ $ret1 == 124 -o $ret2 == 124 ]; then
-			echo "Test timed out."
 			cleanup
 		fi
-		printf -v output "%s%-50s%10s\n" "$output" "$test_exe:" "Fail"
-		fail_count=$((fail_count + 1))
+		printf "%-50s%10s\n" "$test_exe:" "Fail"
+		fail_count+=1
+		[ "$VERBOSE" == "1" ] && print_results "$SO" && print_results "$CO"
```

Comment:
Can we have verbose results for the skipped and passed tests also? May be with a -vv option or something. It would be helpful if there was a way to view all output.